flixel/addons/api/gamejolt/FlxGameJoltRequest.hx

```diff
@@ -0,0 +1,367 @@
+				if (table_id != null)
+					params.push({name: "table_id", value: '$table_id'});
+				if (FlxGameJolt.usertoken != "")
+				{
+					params.push({name: "username", value: FlxGameJolt.username});
+					params.push({name: "user_token", value: FlxGameJolt.usertoken});
+				}
+				else
+					params.push({name: "guest", value: FlxGameJolt.username});
+			case SCORES_GETRANK(sort, table_id):
+				command = "scores";
+				action = "get-rank";
+				params.push({name: "sort", value: '$sort'});
+				if (table_id != null)
+					params.push({name: "table_id", value: '$table_id'});
+			case SCORES_FETCH(fromUser, table_id, limit, betterThan):
+				command = "scores";
+				if (table_id != null)
+					params.push({name: "table_id", value: '$table_id'});
+				if (limit != null)
+					params.push({name: "limit", value: '$limit'});
+				if (betterThan != null)
+					params.push({name: betterThan < 0 ? "worse_than" : "better_than", value: '${Math.abs(betterThan)}'});
+				if (fromUser)
+				{
+					if (FlxGameJolt.usertoken != "")
+					{
+						params.push({name: "username", value: FlxGameJolt.username});
+						params.push({name: "user_token", value: FlxGameJolt.usertoken});
+					}
+					else
+						params.push({name: "guest", value: FlxGameJolt.username});
+				}
+			case SCORES_TABLES:
+				command = "scores";
+				action = "tables";
+			case TROPHIES_FETCH(achieved, trophy_id):
+				command = "trophies";
+				if (achieved != null)
+					params.push({name: "achieved", value: '$achieved'});
+				if (trophy_id != null)
+					params.push({name: "trophy_id", value: '$trophy_id'});
+				params.push({name: "username", value: FlxGameJolt.username});
+				params.push({name: "user_token", value: FlxGameJolt.usertoken});
+			case TROPHIES_ADD(trophy_id):
+				command = "trophies";
+				action = "add";
+				params.push({name: "trophy_id", value: '$trophy_id'});
+				params.push({name: "username", value: FlxGameJolt.username});
+				params.push({name: "user_token", value: FlxGameJolt.usertoken});
+			case TROPHIES_REMOVE(trophy_id):
+				command = "trophies";
+				action = "remove";
+				params.push({name: "trophy_id", value: '$trophy_id'});
+				params.push({name: "username", value: FlxGameJolt.username});
+				params.push({name: "user_token", value: FlxGameJolt.usertoken});
+		}
+		
+		var urlSection:String = '/$command${action != "" ? '/$action' : ""}?game_id=${FlxGameJolt.gameID}${[for (p in params) '&${p.name}=${p.value}'].join("")}';
+		if (signed)
+			urlSection = sign(urlSection).urlEncode();
+		return urlSection;
+	}
+	
+	/**
+	 * Signs a piece of URL according to `usingMd5` parameter of the `FlxGameJolt` class.
+	 * @param   url  The old URL piece.
+	 * @return  The new URL piece.
+	 */
+	function sign(url:String):String
+	{
+		var urlToEncode:String = url + FlxGameJolt.gameKey;
+		return '$daUrl&signature=${FlxGameJolt.usingMd5 ? haxe.crypto.Md5.encode(urlToEncode) : haxe.crypto.Sha1.encode(urlToEncode)}';
```

Comment:
I'm getting a compile error here, seems you forgot to change this, though, when I change `daUrl` to `url` I still seem to get an error message from the server:
> The signature you entered for the request is invalid.

It was really hard to debug this issue, as the program didn't seem to broadcast useful any errors to me. The username and password I entered were correct, so I assume the problem is in your api setup